dev/docker-compose.dev.yml

```diff
@@ -19,6 +19,7 @@ services:
             - ../global:/app/global
             - ../scripts:/app/scripts
         environment:
+            - OPENTELEMETRY_ENABLED=0
```

Comment:
Isn't the default value `false` anyway? Then I would just remove it from the dc file so that it doesn't become even more bloated.